README.md

```diff
@@ -2,8 +2,24 @@
 Holds generated but persistent results from building the `ansible` community package.  This information
 may be referred to by other projects and scripts.
 
+## Milestones
+
 Release engineers check the [milestones](https://github.com/ansible-community/ansible-build-data/milestones) for corresponding releases some time before releasing the package sufficient to solve all related issues.
 
+## Blockers
+
+In the context of the Ansible Community package release workflow, a **release blocker** is a situation which does not allow the package to be released.
+It might come with a new ``ansible-core`` release and affect many of the included collections or in any other way severe affect consistent work of the ``ansible`` package.
```

Comment:
```suggestion
It might come with a new ``ansible-core`` release and affect many of the included collections or in any other way severely affect consistent work with the ``ansible`` package.
```

English isn't my first language, but would this be better?

Reply:
sure:) committed, thanks for spotting:)
i think one thing is `s` shouldn't be there as it's imo a part of `might ...` construction